### ISSUE TITLE
Plot pipeline step fails

### ISSUE THREAD

**Issue by bitkeeper** [HEnquist/camillagui#23]
Hi Henrik,

First thanks for creating CamillaDSP including the surrounding tools.
I have small issue with the plotting in camillagui, hope you can help out:

On press plot of pipeline step:

![image](https://user-images.githubusercontent.com/1525169/107783113-2e7ec700-6d4a-11eb-94c0-1380c404bc2e.png)

Expected a bandwidth/phase plot or something like that, but got:
![image](https://user-images.githubusercontent.com/1525169/107783152-39395c00-6d4a-11eb-9e91-6804731f80d6.png)

After inspection no image is returned, but json:
```
{"name": "Filterstep 1", "samplerate": 44100, "f": [1.0, 1.0099994381708088, 1.0200988651053495, 1.0302992806350826, 
1.040601694589222, 1.0510071268947057, 1.0615166076771687, 1.0721311773629234, 1.0828518867819603, 
1.0936797972719803, 1.1046159807834641, 1.1156615199857958, 1.1268175083744443, 1.1380850503792193, 
1.1494652614736083, 1.1609592682852061, 1.1725682087072515, 1.1842932320112758, 1.19613549896088, 
1.2080961819266487, 1.2201764650022144, 1.2323775441214802, 1.244700627177016, 1.2571469341396395, 
1.2697176971791908, 1.2824141607865158, 1.2952375818966704, 1.308189230013354, 1.3212703873345906, 
1.3344823488796633, 1.3478264226173213, 1.3613039295952656, 1.3749162040709326, 1.3886645936435829, 
1.4025504593877134, 1.4165751759878003, 1.4307401318743929, 1.4450467293615656, 1.4594963847857463, 
1.4740905286459303, 1.4888306057453002, 1.503718075334258, 1.5187544112548907, 1.533941102086877, 
1.5492796512948572, 1.5647715773772723, 
...
```

On the commandline  `plotcamillaconf ` the lot of the pipeline is succesfull.

**Comment by HEnquist** [comment]
The first implementation of the plotting used svgs generated in the backend. Thid was slow and not very flexible, so I changed it so the backend just generates the data points, and the actual plotting is done in the frontend. It looks as if you have a combination of the new backend with the old frontend. Could you check that you are really using the new frontend? I have had some issues where I made some change that just didn't show up because the browser still used the old version from cache.

**Comment by bitkeeper** [comment]
I'm on the latest heads of both master branches:
- camillagui-backend commit 233bac3a30c4d60f3ebbd2ceb54267d082ed018d  (28 Jan)
- camillagui commit d28018e7c77dd7f0edca7c57df763d3d3dadd8a6 (29 Jan)

Tried even a browser I never had used with camillagui before.

Are there any other things I can try?

**Comment by HEnquist** [comment]
Oh I checked the code and the pipelinestep plotting isn't updated. I thought I had fixed that, must have forgotten (or made some strange git mistake). It won't work the way it is now. I'll fix asap!

**Comment by bitkeeper** [comment]
Thanks!

**Comment by JWahle** [comment]
I also stumbled upon this and fixed it in #26.

**Comment by bitkeeper** [comment]
Confirmed that #26 fixed it.
